CHANGELOG.md

```diff
@@ -4,6 +4,11 @@ FEATURES:
 * r/tfe_workspace_run_task, d/tfe_workspace_run_task: Add `stage` attribute to workspace run tasks. ([#555](https://github.com/hashicorp/terraform-provider-tfe/pull/555))
 * r/tfe_workspace_policy_set: Add ability to attach an existing `workspace` to an existing `policy set`. ([#589](https://github.com/hashicorp/terraform-provider-tfe/pull/589))
 
+* r/tfe_workspace: Added attribute for health assessments (drift detection) setting ([550](https://github.com/hashicorp/terraform-provider-tfe/pull/550))
```

Comment:
This is an exciting feature! I think it would be helpful to include the attribute that was added in the comment, plus use present tense for the verb - something like:

```suggestion
* r/tfe_workspace: Adds `assessments_enabled` to workspaces to enable health assessments (such as drift detection) ([550](https://github.com/hashicorp/terraform-provider-tfe/pull/550))
```

Same suggestion for the other changelog entries ✨